shell/src/main/java/org/apache/accumulo/shell/ShellCompletor.java

```diff
@@ -45,13 +48,16 @@ public ShellCompletor(Token rootToken, Map<CompletionSet,Set<String>> options) {
 
   @Override
   @SuppressWarnings({"unchecked", "rawtypes"})
```

Comment:
These warning suppressions might not make sense anymore, now that you've added the appropriate generics to the candidates list.

Reply:
Removed in the latest commit